style.css

```diff
@@ -0,0 +1,80 @@
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0% 37%;
+}
+
+header {
+  height: 92px;
+  width: 355px;
+  background-color: #3c3a39;
+  padding: 10px;
+  color: white;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 75px;
+}
+
+nav ul li {
+  list-style: none;
+}
+
+nav ul li a {
+  text-decoration: none;
+  color: white;
+}
+
+.headline {
+  width: 375px;
+  height: 590px;
+  background: url("assets/images/background.png");
+  background-repeat: no-repeat;
+  background-position: 230px 230px;
+  background-color: #1c1a19;
+}
+
+.heyThere {
+  color: #ff6800;
+  font-family: "Times New Roman", Times, serif;
+  font-size: 30px;
+  margin-bottom: 10px;
+  margin-left: 30px;
+  line-height: 45px;
+  padding-top: 120px;
+}
+
+.career {
+  color: white;
+  margin-left: 30px;
+  height: 24px;
+  font-size: 20px;
+}
+
+.description p {
+  color: white;
+  margin-left: 30px;
+  width: 324px;
+  margin-top: 15px;
+  line-height: 23px;
+}
```

Comment:
[OPTIONAL] Kindly make sure that you are using the same font as the Figma file. :eyes: